src/js/multiple-choice.js

```diff
@@ -21,3 +21,33 @@ const handleOptionButtonClick = (event) => {
 optionButtons.forEach((button) => {
   button.addEventListener('click', handleOptionButtonClick);
 });
+
+const randomizeAnswers = () => {
+  // grab and group multiple choice items by needs-js and no-js  pairs
+  const multipleChoiceItems = document.querySelectorAll(".cmp-multiple-choice__item");
+  const needsJSNoJSPairs = [];
+  for (const [index, element] of multipleChoiceItems.entries()) {
+    if (index % 2 === 0) {
+      needsJSNoJSPairs.push({
+        needsJS: element,
+        noJS: multipleChoiceItems[index + 1],
+      });
+    }
+  }
+  // randomize
+  const randomizedOrder = new Array(needsJSNoJSPairs.length);
+  needsJSNoJSPairs.forEach((pair) => {
+    let newPosition = -1;
+    do {
+      newPosition = Math.floor(Math.random() * (needsJSNoJSPairs.length));
+    } while (randomizedOrder[newPosition]);
+    randomizedOrder[newPosition] = pair;
+  });
```

Comment:
I took a different approach to randomization in #97 that's a bit simpler and will avoid a possible (though unlikely) case where this loop just keeps going because it keeps returning an index that's already been used. You could do this instead:

```js
const randomizedOrder = needsJSNoJSPairs.sort(() => (Math.random() > 0.5 ? 1 : -1));
```

Or if you updated based on my other comment, it would be:

```js
const randomizedOrder = Array.from(multipleChoiceItems).sort(() => Math.random() > 0.5 ? 1 : -1);
```